packages/subscription-manager/src/components/TabsSwitcher/TabsSwitcher.tsx

```diff
@@ -23,7 +23,7 @@ type TabsSwitcherProps = {
 const getRoute = ( baseRoute: string, path: string ): string => {
 	// This is a temporary redirect to the re-skinned Subscription Management portal.
 	// Once the new "Commnets" and "Sites" views are ready, this will be removed.
-	const temporaryRedirect = [ 'comments', 'sites' ];
+	const temporaryRedirect = [ 'comments' ];
```

Comment:
I think we should keep this out of the PR for now - since we don't want to expose the unfinished Sites view to our users yet (unless I missed something).

Instead, we could add the change to the PR test instructions.

Reply:
That's a good point. I'll do it 👍

Reply:
> Instead, we could add the change to the PR test instructions.

The code change should not be required to test. We can use the direct path: http://calypso.localhost:3000/subscriptions/sites

Reply:
Yes, that's how I have the test instructions.